app/migrations/Version20180108120001.php

```diff
@@ -0,0 +1,299 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Symfony\Component\Filesystem\Filesystem;
+
+class Version20180108120001 extends AbstractMigration
+{
+    private const BATCH = 500;
+    private const TMP_REQUESTS = '/tmp/procuration_requests.txt';
+    private const TMP_PROXIES = '/tmp/procuration_proxies.txt';
+
+    private const REQUESTS_TABLE = 'procuration_requests';
+    private const PROXIES_TABLE = 'procuration_proxies';
+    private const REQUEST_JOIN_FIELD = 'procuration_request_id';
+    private const PROXY_JOIN_FIELD = 'procuration_proxy_id';
+    private const JOIN_FIELDS = [
+        self::REQUESTS_TABLE => self::REQUEST_JOIN_FIELD,
+        self::PROXIES_TABLE => self::PROXY_JOIN_FIELD,
+    ];
+    private const PRESIDENTIAL_FIRST_ROUND = 'election_presidential_first_round';
+    private const PRESIDENTIAL_SECOND_ROUND = 'election_presidential_second_round';
+    private const LEGISLATIVE_FIRST_ROUND = 'election_legislative_first_round';
+    private const LEGISLATIVE_SECOND_ROUND = 'election_legislative_second_round';
+
+    private $roundIds = [];
+
+    public function preUp(Schema $schema)
+    {
+        // Save legacy data
+        $fs = $this->warmUpTmp();
+        $offset = -1;
+        while ($requests = $this->selectLegacyRounds(self::REQUESTS_TABLE, ++$offset)) {
+            $fs->appendToFile(self::TMP_REQUESTS, \serialize($requests)."\n");
+        }
+        $offset = -1;
+        while ($proposals = $this->selectLegacyRounds(self::PROXIES_TABLE, ++$offset)) {
+            $fs->appendToFile(self::TMP_REQUESTS, \serialize($requests)."\n");
+        }
+    }
+
+    public function up(Schema $schema)
+    {
+        $this->addSql('CREATE TABLE procuration_requests_to_election_rounds (procuration_request_id INT NOT NULL, election_round_id INT NOT NULL, INDEX IDX_A47BBD53128D9C53 (procuration_request_id), INDEX IDX_A47BBD53FCBF5E32 (election_round_id), PRIMARY KEY(procuration_request_id, election_round_id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE procuration_proxies_to_election_rounds (procuration_proxy_id INT NOT NULL, election_round_id INT NOT NULL, INDEX IDX_D075F5A9E15E419B (procuration_proxy_id), INDEX IDX_D075F5A9FCBF5E32 (election_round_id), PRIMARY KEY(procuration_proxy_id, election_round_id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE elections (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, introduction LONGTEXT NOT NULL, place VARCHAR(255) DEFAULT NULL, UNIQUE INDEX UNIQ_1BD26F335E237E06 (name), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
```

Comment:
You removed the field `place` from entity. You should not create this field here, should you ?

Reply:
Good catch, thx!

Reply:
Fixed.